Review.java

```diff
@@ -165,4 +165,30 @@ public static String randomAdjective()
       return randomNegativeAdj();
     }
   }
+   public static int starRating(String fileName)
+   {
+   

```

Comment:
Maybe add a comment explaining why you split the ratings up like this.

Reply:
Thank you! I added some comments, hope it explains it!